modules/assim.batch/R/pda.emulator.R

```diff
@@ -145,9 +145,8 @@ pda.emulator <- function(settings, params.id=NULL, param.names=NULL, prior.id=NU
   }
   
   # define range to make sure mcmc.GP doesn't propose new values outside 
-  rng=matrix(c(sapply(prior.fn$qprior[prior.ind] ,eval,list(p=0)),
-        sapply(prior.fn$qprior[prior.ind] ,eval,list(p=1))),
-        nrow=n.param)
+  
+  rng=matrix(c(apply(X,2,min), apply(X ,2,max)),nrow=n.param)
```

Comment:
We can discuss later -- pros and cons of both, but until you can tweak the emulators to regress to the tails instead of the mean I think we're better off with your edit.